osu.Game/Screens/Play/GameplayClockContainer.cs

```diff
@@ -63,17 +62,17 @@ public class GameplayClockContainer : Container
 
         private readonly FramedOffsetClock platformOffsetClock;
 
-        public GameplayClockContainer(WorkingBeatmap beatmap, IReadOnlyList<Mod> mods, double gameplayStartTime)
+        public GameplayClockContainer([NotNull] ITrack track, WorkingBeatmap beatmap, IReadOnlyList<Mod> mods, double gameplayStartTime)
         {
             this.beatmap = beatmap;
             this.mods = mods;
             this.gameplayStartTime = gameplayStartTime;
+            this.track = track;
```

Comment:
In this usage, is there a reason we can't just call `GetTrack()` on the `WorkingBeatmap`, since it will return the same instance?

Reply:
Yeah, probably fine.